airflow/providers/databricks/hooks/databricks_base.py

```diff
@@ -248,9 +248,11 @@ def _get_aad_token(self, resource: str) -> str:
                         azure_ad_endpoint = self.databricks_conn.extra_dejson.get(
                             "azure_ad_endpoint", AZURE_DEFAULT_AD_ENDPOINT
                         )
+                        azure_databricks_proxy = self.databricks_conn.extra_dejson.get("databricks_proxy")
```

Comment:
I would go with simpler name - `proxies`, or something like that.

Reply:
Thanks for taking the time to review. I have gone with your suggestions